src/index.js

```diff
@@ -1,172 +1,105 @@
-const { mapKeys } = require('lodash')
 const core = require('@actions/core')
-const lighthouse = require('lighthouse')
-const { getFilenamePrefix } = require('lighthouse/lighthouse-core/lib/file-namer')
-const chromeLauncher = require('chrome-launcher')
-const { ensureDir } = require('fs-extra')
-const { join } = require('path')
-const { writeFile } = require('fs').promises
-const { readFileSync } = require('fs')
-
-// audit urls with Lighthouse
+const childProcess = require('child_process')
+const lhciCliPath = require.resolve('@lhci/cli/src/cli.js')
+const input = require('./input.js')
```

Comment:
let's move input parsing to the `main`, because if these steps fail, the error will not be integrated into action workflow.

Reply:
I think the errors show up as expected? Since it fails with `core.setFailed` and immediately process exits with `1` I think that this fails the same as if it was in main. Is there something that isn't integrated this way?

![image](https://user-images.githubusercontent.com/6392995/68559383-a6e13480-03f1-11ea-926e-011236feb168.png)